microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+            "Print to PDF succeeded" : "Print to PDF failed";
+        MessageBox.Show(this, message, "Print To PDF Completed");
+    }
+}
+```
+
+**APIs:**
+
+* [CoreWebView2.PrintToPdfAsync(resultFilePath, printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfasync#microsoft-web-webview2-core-corewebview2-printtopdfasync)
+* [CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+```cpp
+void FileComponent::PrintToPdf(bool enableLandscape)
+{
+    WCHAR defaultName[MAX_PATH] = L"WebView2_PrintedPdf.pdf";
+    OPENFILENAME openFileName = CreateOpenFileName(defaultName, L"PDF File\0*.pdf\0");
+    if (GetSaveFileName(&openFileName))
+    {
+        wil::com_ptr<ICoreWebView2PrintSettings> printSettings = nullptr;
+        if (enableLandscape)
+        {
+            wil::com_ptr<ICoreWebView2Environment5> webviewEnvironment5;
+            CHECK_FAILURE(m_appWindow->GetWebViewEnvironment()->QueryInterface(
+                IID_PPV_ARGS(&webviewEnvironment5)));
+            if (webviewEnvironment5)
+            {
+                CHECK_FAILURE(webviewEnvironment5->CreatePrintSettings(&printSettings));
+                CHECK_FAILURE(
+                    printSettings->put_Orientation(COREWEBVIEW2_PRINT_ORIENTATION_LANDSCAPE));
+            }
+        }
+
+        wil::com_ptr<ICoreWebView2_7> webview2_7;
+        CHECK_FAILURE(m_webView->QueryInterface(IID_PPV_ARGS(&webview2_7)));
+        if (webview2_7)
+        {
+            m_printToPdfInProgress = true;
+            CHECK_FAILURE(webview2_7->PrintToPdf(
+                openFileName.lpstrFile, printSettings.get(),
+                Callback<ICoreWebView2PrintToPdfCompletedHandler>(
+                    [this](HRESULT errorCode, BOOL isSuccessful) -> HRESULT {
+                        CHECK_FAILURE(errorCode);
+                        m_printToPdfInProgress = false;
+                        auto showDialog = [isSuccessful] {
+                            MessageBox(
+                                nullptr,
+                                (isSuccessful) ? L"Print to PDF succeeded"
+                                               : L"Print to PDF failed",
+                                L"Print to PDF Completed", MB_OK);
+                        };
+                        m_appWindow->RunAsync([showDialog]() { showDialog(); });
+                        return S_OK;
+                    })
+                    .Get()));
+        }
+    }
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_7::PrintToPdf(resultFilePath, printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_7#printtopdf)
+* [ICoreWebView2PrintSettings interface](microsoft-edge/webview2/reference/win32/icorewebview2printsettings)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## PrintToPdfStream
```

Comment:
```suggestion
## The PrintToPdfStream method
```
the reason i suggest this is because there's a problematic case, the Print method, which makes a poor 1-word heading, "Print" b/c that heading is ambiguous re: verb vs. API item name.